examples/spouse_example/piggy_extractor/application.conf

```diff
@@ -0,0 +1,56 @@
+deepdive {
+
+  db.default {
+    driver: "org.postgresql.Driver"
+    url: "jdbc:postgresql://"${PGHOST}":"${PGPORT}"/"${DBNAME} # "
+    user: ${PGUSER}
+    password: ${PGPASSWORD}
+    dbname: ${DBNAME}
+    host: ${PGHOST}
+    port: ${PGPORT}
+  }
+
+  pipeline.run: "all"
+  pipeline.pipelines.all: [
+    "ext_clear_table",
+    "ext_people",
+    "ext_spouses",
+  ]
+
+  extraction.extractors {
+
+    ext_clear_table {
+      style: "sql_extractor"
+      sql: """
+        DELETE FROM people_mentions;
+        DELETE FROM has_spouse;
+        DELETE FROM has_spouse_features;
+        """
+    }
+
+    ext_people {
+      input: """
+          SELECT  sentence_id,
+                  words,
+                  ner_tags
+          FROM    sentences
+          """
+      output_relation: "people_mentions"
+      udf_dir: ${APP_HOME}"/udfs"
+      udf: "python people.py"
+      dependencies: ["ext_clear_table"]
+      style: "piggy_extractor"
+    }
+
+    ext_spouses {
+      input: "people_mentions"
+      output_relation: "has_spouse(sentence_id, person1_id, person2_id)"
+      udf_dir: ${APP_HOME}"/udfs"
```

Comment:
It seems `udf_dir` is what's being shipped by piggy, right?  A comment around this line would be extremely helpful if so.